recipes/eglibc/eglibc_2.18-r24943.oe

```diff
@@ -3,9 +3,18 @@ require ${PN}.inc
 SRC_REV := "${@d.get('PV').split('-',1)[1]}"
 PV := "${@d.get('PV').split('-',1)[0]}"
 
+RECIPE_FLAGS:machine	+= " eglibc_powerpc_32bit_addr"
```

Comment:
Why not just reuse the powerpc_32bit_addr USE flag?  Would there be any situations where it would be relevant to enable only one of them?